### DISCUSSION TITLE
upgrade to 1.2.11 problem

### DISCUSSION THREAD

**Discussion by chiabetatester** [Chia-Network/chia-blockchain#9628] [Support]
hi there i upgraded from 1.2.6 (using this for months) to 1.2.11
after requesting to put a paraphrase in , when the app ran all i saw was spinning circles, it never synced.
so i closed the chia program and restarted it.

but after entering paraphrase in , i got the error:
'NoneType' object is not subscriptable.
i click ok to this then i get another screen saying:
fingerprint ok.
i click the ok but nothing happens

now when i restart the chia app
it asks for the 24word .
after i put that in i click skip to the restore metadata
i get the error above :
'NoneType' object is not subscriptable.
followed by the fingerprint error

i can't get past this window to skip the restore metadata !

help

**Comment by adspx5** [comment]
Stop chia. delete db and walletdb. let db resync from the start. All problems will gone.
It's a headache, but only this helped me few times.
After that start making db backup each 10 days ;)

**Comment by Jacek-ghub** [comment]
There are basically only two things that can get in your way.  First is databases (blockchain and wallet) and the other is config.yaml.  More or less, only those two things are potentially version specific.

Also, there is a new folder in your root directory (something like .chia_keys or so).  That is more or less a useless folder.  It basically reflects your new password.  When you delete that folder, your password is gone, and you can add a new one.  I would first try to kill that folder, as the operation is harmless.

If that doesn't work, I would rename your current config.yaml, and move it aside, restart chia, and you will get a new config.yaml that will be properly populated.  If you made some changes to the original one, restore those changes (e.g., list of plot folders, or paths to your dbs).

If that doesn't work, killing the wallet db would be the next step.  Restoring that db takes "only" few hours.

If you still cannot start, the blockchain db is most likely the problem.  Move your existing aside first.  Then you have two options.  The first is to let your box resync, but that will take few days (two/three - partially depending on your box CPU).  A faster option, and potentially worth to try is downloading blockchain db from chia-database.com.  This is a bit controversial option, but you would be up and running in about an hour (excluding your wallet sync).

Of course, there is also a possibility that there is something not right with your system, but that is more difficult to nail down.   For that reason, downloading that db as the first step (just to test whether that is the case) is maybe another good option to start.  If it works, you can decide whether you want to stay with that downloaded db, or kill it and fully sync.

Again, I am not endorsing that db, and have nothing to do with that site.  I do understand that it could be poisoned, and that was a bit discussed on chia forum.  My understanding is that the guy has tried to legitimize that process (offering download bandwidth to legit db source - e.g., chia or some bigger pool), but I am not really sure about that, whether he started the process or how far it went.  There were some discussions about getting a daily snapshots from Flex Pool,.  But again, as far as I know, it oculd be poisoned, but it is at least worth to try to exclude potential db corruption problems.

**Comment by chiabetatester** [comment]
thanks for the reply but in the end i didnt have do any of the above !
i gave up trying to to get 1.2.11 to work. i downloaded 1.2.10 installed it and it updated and ran properly as it should

it logged in and sync'ed straight away so i think there is a problem with 1.2.11 if someone else runs into my issue.

**Reply by Jacek-ghub** [reply]
Thank you for the update!!  I tried to upgrade to v1.2.11 today and wasted about 6 hours trying to get that shit working.  The only thing that I was getting was the big spinner, and a text that it is trying to connect to a wallet.  There were no logs whatsoever.  I don't know how they expect people to troubleshoot it, measure the thickness of that spinner?

I followed your suggestion, installed v1.2.10, and it started working in about 20 minutes.  What a joke.

I will be looking to switch to Flex farmer, as this code is just plain joke.

**Reply by Jacek-ghub** [reply]
@chiabetatester I have tried to update again to v1.2.11 (this time from v1.2.10 that I installed last time).  It updated without any problems.  It may be that v1.2.10 needs to do some changes (maybe to .chia_keys ???) in order to get v1.2.11 working.
